AK/JsonObjectSerializer.h

```diff
@@ -129,6 +129,16 @@ class JsonObjectSerializer {
         return {};
     }
 
+    ErrorOr<void> add(StringView key, float value)
+    {
+        TRY(begin_item(key));
+        if constexpr (IsLegacyBuilder<Builder>)
+            TRY(m_builder.try_appendff("{}", value));
+        else
+            TRY(m_builder.appendff("{}", value));
+        return {};
+    }
+
```

Comment:
Please put the `float` code always next to the double code and into the KERNEL guard

Reply:
After further investigation I fully agree. I have removed this interface and kept only the `to_float()` and `ErrorOr<void> add(StringView key, float value)` and moved them into the kernel guards.